src/home/HomePage.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class='homepage'>
+    <tasks-list :taskList='taskList' v-on:complete='onComplete($event)' />
+    <completed-tasks-list
+      :taskList='taskList'
+      v-on:complete='onComplete($event)'
+      v-on:delete='onDelete($event)'
+    />
+  </div>
+</template>
+
+<script>
+  import CompletedTasksList from './components/CompletedTasksList.vue';
+  import TasksList from './components/TasksList.vue';
+
+  export default {
+    data() {
+      return {
+        taskList: {},
+      };
+    },
+
+    components: { TasksList, CompletedTasksList },
+    created() {
+      this.fetchTasks();
+    },
+    methods: {
+      fetchTasks() {
+        if ('taskList' in localStorage) {
+          console.log('found tasklist in localstorage');
+          this.taskList = JSON.parse(localStorage.getItem('taskList'));
+        } else {
+          const taskList = {
+            IT: [],
+            HR: [],
+            'New Hire Paperwork': [],
+            'Culture Orientation': [],
+            Other: [],
+          };
+          // {'IT':[{'title':'Do testing','assignee':'Sahil','date':'2022-08-17'}],'HR':[{'title':'Schedule a meeting','assignee':'Michael','date':'2022-08-24'}],'New Hire Paperwork':[],'Culture Orientation':[],'Other':[]}
+          localStorage.setItem('taskList', JSON.stringify(taskList));
+          this.taskList = JSON.parse(localStorage.getItem('taskList'));
+
+          console.log(JSON.parse(localStorage.getItem('taskList')));
+        }
+      },
+      onComplete(event) {
+        console.log(event);
```

Comment:
Evenever, we are writing any console then we should describe the variable that we are logging.
like `console.log('event' + event)`.

Reply:
ok, will remember